drivers/ovsSwitch.go

```diff
@@ -363,17 +365,14 @@ func (sw *OvsSwitch) UpdatePort(intfName string, cfgEp *mastercfg.CfgEndpointSta
 }
 
 // DeletePort removes a port from OVS
-func (sw *OvsSwitch) DeletePort(epOper *OvsOperEndpointState) error {
+func (sw *OvsSwitch) DeletePort(epOper *OvsOperEndpointState, nwType string) error {
 
```

Comment:
Avoid passing the nwType if possible. Instead, use a more context relevant argument.